library/Ivoz/Api/Csv/Serializer/Mixer/CsvMixer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Ivoz\Api\Csv\Serializer\Mixer;
+
+use Ivoz\Api\Core\Serializer\Mixer\MixerInterface;
+
+class CsvMixer implements MixerInterface
+{
+    const FORMAT = 'csv';
+
+    public function __construct()
+    {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsFormat($format = null)
+    {
+        return static::FORMAT === $format;
```

Comment:
```static:: will not be resolved using the class where the method is defined but it will rather be computed using runtime information```

http://www.php.net/manual/en/language.oop5.late-static-bindings.php